.github/workflows/release.yml

```diff
@@ -0,0 +1,17 @@
+name: Publish release on github
+on:
+  workflow_dispatch:
```

Comment:
Does this workflow need to be run manually?

Reply:
yes, once we tag the release then we have to run this manually

Reply:
can't this be triggered to run when a tag is pushed?